.github/workflows/build-commit.yaml

```diff
@@ -2,26 +2,38 @@ name: build-commit
 
 on:
   workflow_dispatch:
-  workflow_call:
-    secrets:
-      ACTIONS_AWS_ROLE_ARN:
-        description: The ARN of the AWS role to assume
+    inputs:
+      arch:
+        type: choice
+        options:
+        - x86
+        - arm
+        description: The machine architecture to build for
         required: true
-    outputs:
-      wheel:
-        description: The wheel file that was built
-        value: ${{ jobs.build-commit.outputs.wheel }}
+        default: x86
```

Comment:
Should this be a choice, or should we just make it a matrix and build both?

Reply:
Could be a matrix, and we could provide "checkboxes" to the end user to turn on/off each variant of the matrix. However, the implementation of that would just be very long and verbose... May be more productive to merge in a following PR later?